Tasks/PackerBuildV1/task.json

```diff
@@ -1,20 +1,20 @@
 {
-    "id": "845fd4f4-642d-4694-8514-047948a5a556",
-    "name": "PackerBuild",
-    "friendlyName": "Build Machine Image",
-    "description": "Build machine image using Packer. This image can be used for Azure Virtual machine scale set deployment",
+    "id": "14e5851b-8644-4146-ac85-5c8bef5a3efc",
```

Comment:
I think this was required for your testing. Kindly revert the `id` `name` `friendlyName` `description` `author` `major` and `patch => 4`
Also make sure you build the task after theses changes so that the same are reflected in resources.resjson and task.loc.json file

Reply:
forgot to stage a change, good catch, updated.